tests/tests_geomstats/test_visualization_grassmannian.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for visualization."""
+import os
+import sys
+import warnings
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+import tests.conftest
+from geomstats.visualization.grassmannian import Grassmannian
+
+sys.path.append(os.path.dirname(os.getcwd()))
+# TODO: Change these paths with the corresponding paths on your computer
+
+print(os.getcwd())
+warnings.filterwarnings("ignore")
+
+matplotlib.use("Agg")  # NOQA
+
+
+class TestVisualizationGrassmanian(tests.conftest.TestCase):
```

Comment:
the functions:
-  `projection_to_two_d`
-  `projection_to_three_d`
should be tested as well

Reply:
and: `two_d_to_projection`